### DISCUSSION TITLE
abs-proxy-base-path didn't work

### DISCUSSION THREAD

**Discussion by yuzhuoheng** [coder/code-server#7035] [Q&A]
abs-proxy-base-path   It didn't work. This is my startup command:
![企业微信截图_17288752683468](https://github.com/user-attachments/assets/89b1e5e0-726b-4b52-b572-8af7436bc9be)
I tried to open it with a browser, and the page prompt was as follows:
![企业微信截图_1728875413984](https://github.com/user-attachments/assets/cd271829-8589-4194-8425-e28be1903222)

**Comment by code-asher** [comment]
Just leaving a note for anyone following along that we are discussing this in a few other places.

**Reply by yuzhuoheng** [reply]
Thank you very much. I have reviewed the document（https://github.com/coder/code-server/blob/main/docs/guide.md#prefixing-absproxyport-with-a-path） you recommended. But I don't know where to put the svelte.config.js you recommended (I deployed the code server internally in Docker)
![企业微信截图_17291313482047](https://github.com/user-attachments/assets/4920f291-0dc8-4c54-bd6c-1104d2c8391b)

**Reply by code-asher** [reply]
I have not used Svelte so I am not sure where it goes, but my guess is that you put it at the root of your project.
